rust/capture/src/limiters/token_dropper.rs

```diff
@@ -0,0 +1,78 @@
+use std::collections::HashMap;
+
+use tracing::warn;
+
+#[derive(Default)]
+pub struct TokenDropper {
+    to_drop: HashMap<String, Vec<String>>,
+}
+
+impl TokenDropper {
+    // Takes "<token>:<distinct_id or *>,<distinct_id or *>;<token>..."
```

Comment:
Feels like for now the separator should be a comma to match the others. If we prefer semicolon then we should swap them all to semicolon as otherwise someone will for sure make a mistake.

Reply:
Will do in a follow-up pr